maritalk/resources/api.py

```diff
@@ -28,11 +28,13 @@ def __str__(self):
 
 
 class MariTalk:
-    def __init__(self, key: str, api_url: str = "https://chat.maritaca.ai/api"):
+    def __init__(self, key: str, api_url: str = "https://chat.maritaca.ai/api", model="MariTalk"):
```

Comment:
I think I prefer lowercase here, WDYT?

```suggestion
    def __init__(self, key: str, api_url: str = "https://chat.maritaca.ai/api", model="maritalk"):
```